### PR TITLE
Create LIT6943PredicationsChr

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a record according to our discussion in https://github.com/BetaMasaheft/Documentation/issues/2490.

### CODE REVIEW

File: new/LIT6943PredicationsChr.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-02-28">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="ecrm:P129i_is_subject_of" active="LIT6943PredicationsChr" passive="PRS5684JesusCh"/>
```
Comment (DenisNosnitsin1970) [inline new/LIT6943PredicationsChr.xml:57]: I would add a relation to the ms where the text is attested.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT6943PredicationsChr.xml (hunk 1 above)
Comment (eu-genia) [reply new/LIT6943PredicationsChr.xml:57]: Denis there is already the `witness `in `ListWit`, we use `witness` for attestations of texts

File: new/LIT6943PredicationsChr.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6943PredicationsChr" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Predications about the birth of Christ made by the Prophets</title>
```
Comment (eu-genia) [inline new/LIT6943PredicationsChr.xml:8]: en

File: new/LIT6943PredicationsChr.xml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="ecrm:P129i_is_subject_of" active="LIT6943PredicationsChr" passive="PRS5684JesusCh"/>
+                </listRelation>
+            </div><!---->
+            <div type="edition" subtype="textsubunit">
```
Comment (eu-genia) [inline new/LIT6943PredicationsChr.xml:60]: it should be 
```
<div type="edition" xml:lang="gez">
 <div type="textpart">
 <ab>your text here</ab>
</div>
</div>
```
there is potentially the edition of the entire text, and nested there is a textpart. textsubunit would be a clearly marked chapter or paragraph, I do not think this is the case.

File: new/LIT6943PredicationsChr.xml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianContent"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="OldTestament"/>
```
Comment (eu-genia) [inline new/LIT6943PredicationsChr.xml:41]: OldTestament is for biblical books for the OT, remove

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT6943PredicationsChr.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/LIT6943PredicationsChr.xml:8]: "Predications" is smth different, as far as I can see. I assume Strelcyns title can be taken over.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

**Pull request merged**